release/release.sh

```diff
@@ -85,10 +85,10 @@ case $1 in
     MSG="\"Release: $VERSION for QIX schema version $QIX_VERSION\""
```

Comment:
Do we still need the escaped quotes in `MSG` now? The different ways we expand variables in this script is a bit inconsistent :)